### ISSUE TITLE
Update window messages show strange text

### ISSUE THREAD

**Issue by newportsolutionsgroup** [timanrebel/HockeyApp#18]
Tim, great work on the HockeyApp module. 
Have you or anyone had the same issue I keep facing where the strings in the messages do not display correctly?

I was told it was a problem with the HockeySDKResources.bundle but it is included and present in your module. 
Take a look at the screen shot. Am I missing something?

![simulator screen shot sep 1 2016 12 26 22 pm](https://cloud.githubusercontent.com/assets/1017169/18181278/585a6848-703f-11e6-9d8a-2c011a1aad93.png)

**Comment by kopiro** [comment]
Any update here?
I tried too compile myself chaings paths, but with no luck :(

**Comment by timanrebel** [comment]
Could you copy the .bundle file to the app/platforms/iphone directory and try again?

**Comment by newportsolutionsgroup** [comment]
Yes - Resolved. I got this working in Appcelerator 5.4GA in both Alloy and Classic (which is what I needed) with some tips from @timanrebel . First I updated the HockeySDK and recompiled / packaged the iOS module (but this step is not mandatory)

The trick is to place the HockeySDKResources.bundle in your mobile app project folder.

Using ALLOY: Copy the bundle to: app\assets\iphone\HockeySDKResources.bundle
Using Classic: Copy the bundle folder to: Resources\iphone\HockeySDKResources.bundle

*Clean your project and rebuild. - Presto!

**Comment by kopiro** [comment]
It's possible that this doesn't works with 5.2.2.GA?

**Comment by kopiro** [comment]
![image](https://cloud.githubusercontent.com/assets/839700/18440807/c5b81664-790a-11e6-916d-47c531c15c96.png)

**Comment by kopiro** [comment]
Same thing with 5.4.0.GA:

[DEBUG] [HockeySDK] -[BITCrashManager initWithAppIdentifier:appEnvironment:hockeyAppClient:]/252 [HockeySDK] WARNING: HockeySDKResources.bundle is missing, will send reports automatically!

**Comment by newportsolutionsgroup** [comment]
Clean and rebuild? Did you remove the bundle when you compiled the module?

**Comment by kopiro** [comment]
Yes.

**Comment by timanrebel** [comment]
@kopiro Not `app/assets`, but `app/platform` afaik. Did it work in `app/assets` for you @newportsolutionsgroup ?

**Comment by newportsolutionsgroup** [comment]
Yes it worked for me in app/assets

**Comment by Wingee** [comment]
I've had the same issue. Sorted it but it keeps popping up with Romanian text! Any ideas how to set the locale correctly for English? I can't see how
